src/react/databrowser/objects/details/Properties.jsx

```diff
@@ -61,14 +67,53 @@ function Properties({ objectMetadata }: Props) {
               <Clipboard text={objectMetadata.eTag} />{' '}
             </T.ExtraCell>
           </T.Row>
+          <T.Row>
+            <h3> Data Protection </h3>
+          </T.Row>
           <T.Row>
             <T.Key> Lock </T.Key>
             <T.Value>
-              {objectMetadata.lockStatus === 'LOCKED' && <>Locked ({objectMetadata.objectRetention.mode.toLowerCase()})<br />until {objectMetadata.objectRetention.retainUntilDate}</>}
-              {objectMetadata.lockStatus === 'RELEASED' && `Released - since ${objectMetadata.objectRetention.retainUntilDate}`}
+              {objectMetadata.lockStatus === 'LOCKED' && (
+                <>
+                  Locked ({objectMetadata.objectRetention.mode.toLowerCase()})
+                  <br />
+                  until {objectMetadata.objectRetention.retainUntilDate}
+                </>
```

Comment:
I think it is missing the locked icon here for the discoverability of the icon meaning (which was the starting point of this refactoring

Reply:
I've seen it afterwards in commit https://github.com/scality/zenko-ui/pull/214/commits/540df519b27ffa9a9e66cf2d34a0a85fdb9afd9b